src/ApiService/ApiService/onefuzzlib/orm/Orm.cs

```diff
@@ -178,9 +180,12 @@ public class StatefulOrm<T, TState, TSelf> : Orm<T>, IStatefulOrm<T, TState> whe
                     continue;
                 }
 
-                throw new Exception($"State transition method '{state}' in '{thisType?.Name}' does not have the correct signature. Expected '{delegateSignature}'  actual '{methodInfo}' ");
+                throw new InvalidOperationException($"State transition method '{state}' in '{thisType.Name}' does not have the correct signature. Expected '{delegateSignature}'  actual '{methodInfo}' ");
             };
 
+            if (missing.Any()) {
```

Comment:
Not all state have a corresponding transition method.
ex: JobState.Enabled and JobState.Stopped do not have a transition method in jobs.py

Reply:
> Not all state have a corresponding transition method. ex: JobState.Enabled and JobState.Stopped do not have a transition method in jobs.py

I think that it's probably a good idea that we explicitly have these as no-op transitions, so we can check them all?

Reply:
sounds good